stock_inventory_cost_info/models/stock_inventory.py

```diff
@@ -17,6 +17,6 @@ class InventoryLine(models.Model):
     @api.depends("product_qty", "theoretical_qty", "inventory_id.state")
     def _compute_adjustment_cost(self):
         for record in self:
-            adjusted_qty = record.product_qty - record.theoretical_qty
-            adjustment_cost = adjusted_qty * record.product_id.standard_price
-            record.adjustment_cost = adjustment_cost
+            record.adjustment_cost = (
+                record.difference_qty * record.product_id.standard_price
```

Comment:
@sergio-teruel The depends should reflect the fields your computation depends on.